src/main/java/seedu/address/logic/commands/DeletePatientCommand.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.commands;
```

Comment:
do we remove deleteCommand after implementing this?

Reply:
Yes the original Delete Command from AB4 will be removed